.github/workflows/inspektor-gadget.yml

```diff
@@ -697,6 +697,53 @@ jobs:
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+  analyze-ig-memory:
+    name: IG binary memory analysis
+    runs-on: ubuntu-latest
+    needs: 
+      - build-ig 
```

Comment:
You could add "build-and-push-gadgets" in order to run `trace_dns:latest` as compiled by the CI instead of downloading it from internet.